tmt/steps/provision/artemis.py

```diff
@@ -675,6 +692,8 @@ def go(self) -> None:
             data=data,
             name=self.name,
             parent=self.step)
+        from tmt.utils import Path
+        self._guest.key = [Path(k) for k in data.key]
```

Comment:
This should be a workaround until a proper fix of https://github.com/teemtee/tmt/issues/2352 lands.

Reply:
Yes, sorry this is indeed a workaround so that's why I'm keeping this PR as a draft.